crates/libp2p-networking/tests/common/mod.rs

```diff
@@ -190,77 +186,33 @@ pub async fn print_connections<K: SignatureKey + 'static>(handles: &[Arc<Network
 pub async fn spin_up_swarms<S: Debug + Default + Send, K: SignatureKey + 'static>(
     num_of_nodes: usize,
     timeout_len: Duration,
-    num_bootstrap: usize,
 ) -> Result<Vec<(HandleWithState<S, K>, NetworkNodeReceiver)>, TestError<S>> {
     let mut handles = Vec::new();
     let mut bootstrap_addrs = Vec::<(PeerId, Multiaddr)>::new();
```

Comment:
if all nodes are bootstrap now, does this make sense for a name? instead of `node_addrs` or something